tests/trainer/test_trainer.py

```diff
@@ -1664,6 +1664,25 @@ def test_pytorch_profiler_nested(tmpdir):
         assert pa[n] == expected_[n]
 
 
+@pytest.mark.skip(reason="This tests need to be run with nvprof")
+@pytest.mark.skipif(torch.cuda.device_count() < 1, reason="test requires a GPU machine")
+def test_pytorch_profiler_nested_emit_nvtx(tmpdir):
+    """
+    How to run this test.
+    nvprof --profile-from-start off -o trace_name.prof -- pytest {}:test_pytorch_profiler_nested_emit_nvtx
+    python -c "import torch;print(torch.autograd.profiler.load_nvprof('{}/trace_name.prof'))"
+    """
+    profiler = PyTorchProfiler(use_cuda=True, emit_nvtx=True)
```

Comment:
this is just about that it does not crash, how you know that all is fine?

Reply:
Great question ! I wanted to see with you if we should run this one in the CI.

Reply:
what you mean to run in CI, we shall test all our code :]